src/interface/src/app/treatments/direct-impacts/metric-filters/metric-filters.component.ts

```diff
@@ -0,0 +1,162 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { MetricSelectorComponent } from './metric-selector/metric-selector.component';
+import { NgFor } from '@angular/common';
+
+export interface Metric {
+  id: string;
+  color: string;
+  label: string;
+  active: boolean;
+}
+
+export const METRICS: Metric[] = [
+  {
+    id: 'ID_CROWN_BULK_DENSITY',
+    color: '#FF5733',
```

Comment:
not sure I understand this, the color will come from the position of the metric not the metric itself, if Im not mistaken

Reply:
My bad, I thought it was defined based on the selected metric. I updated it, you will see the changes in the next commit.